www/components/footer.tsx

```diff
@@ -31,6 +31,17 @@ export function Footer(): JSX.Element {
           Interactors <IconExternal />
         </a>
       </section>
+      <section class="flex flex-col gap-y-1">
+        <h1 class="text-sm uppercase font-bold text-blue-primary dark:text-blue-secondary mb-4">
```

Comment:
Pedantic, but shouldn't there only be one `h1` per page? Seems to be an existing pattern, so appropriate in a follow-up PR.